.github/workflows/gh_release.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will install Python dependencies, run pre-release script, and
+# then bundle a github release
+
+name: Release generator
```

Comment:
That's a good idea. I'll look into it.

Reply:
Upon reading up about actions/workflows/deployment, I don't think this will be feasible. I called this an action, but really it's a workflow, and a workflow doesn't only encode what happens but also when it happens, which a published action doesn't do.

One way to provide centralized reusability could be to use git submodules instead of copying the file around.